lib/cucumber/cli.js

```diff
@@ -89,6 +89,8 @@ function Cli(argv) {
 \n\
 --no-source                   Don't print the source uris.\n\
 \n\
+--[no-]colors                 Enable/disable colors in output.\n\
```

Comment:
That's `--no-color` in Cucumber ruby. Let's stay consistent (even though `--no-colors` is better, imo). Or both would be fine as well.